backend/i18n/backend__keys.json

```diff
@@ -0,0 +1,325 @@
+{
```

Comment:
Do we need this in the code? I mean it is only a temporary file that is used "nl", right?

Reply:
Yea i can leave it out, if we need it we can generate it anyways.